webview/src/experiments/util/buildDynamicColumns.tsx

```diff
@@ -107,16 +107,15 @@ const buildDynamicColumns = (
 
       const childColumns = buildDynamicColumns(properties, path)
 
-      const column: ColumnGroup<Experiment> | TableColumn<Experiment> = {
+      return {
         Cell,
         Header,
         accessor: pathArray && buildAccessor(pathArray),
         columns: childColumns.length > 0 ? childColumns : undefined,
         group: type,
         id: path,
         name: label
-      }
-      return column
+      } as ColumnGroup<Experiment> | TableColumn<Experiment>
```

Comment:
[F] Another type error, I have manually tested that the table works as expected but I'd appreciate everyone trying it out.

Reply:
same for plots